game/scripts/vscripts/Sections/Chapter4/SectionOpening.ts

```diff
@@ -160,6 +185,7 @@ function checkIfScanCoversTheLocation(targetScanLocation: Vector): boolean {
             return true;
         }
         displayDotaErrorMessage("Scan the required location");
+        // TODO: Play opening_15 audio SL
```

Comment:
Let's put it in, we should have the audio by now.

Reply:
It's in now.